templates/pager.html

```diff
@@ -0,0 +1,16 @@
+<html>
+<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+        
```

Comment:
Since most of the head are same, you can probably create a layout and import that rather than writing the same thing and reduce the redundancy of code.